### PR TITLE
Add elastic credentials to calls + handle exception

### PR DESCRIPTION
**Pull request by samleeflang**
Needed for the new setup of elasticsearch with authentication.
Previous elastic failed silently as we didn't capture the exception, added the exception handling

### CODE REVIEW

File: src/main/java/eu/dissco/core/digitalspecimenprocessor/service/ProcessingService.java [hunk 1]
```diff
@@ -337,7 +340,7 @@ private Set<DigitalSpecimenRecord> createNewDigitalSpecimen(List<DigitalSpecimen
       }
       log.info("Successfully created {} new digitalSpecimen", digitalSpecimenRecords.size());
       return digitalSpecimenRecords.keySet();
-    } catch (IOException e) {
+    } catch (IOException | ElasticsearchException e) {
```
Comment (southeo) [inline src/main/java/eu/dissco/core/digitalspecimenprocessor/service/ProcessingService.java:343]: Should you not apply the same logic to updates?

**Review by southeo — COMMENTED** [review]
Looks good, verify you don't need to do this with updates too.

File: src/main/java/eu/dissco/core/digitalspecimenprocessor/service/ProcessingService.java (hunk 1 above)
Comment (samleeflang) [reply src/main/java/eu/dissco/core/digitalspecimenprocessor/service/ProcessingService.java:343]: Added them at the other places.

**Review by southeo — APPROVED** [review]
:1st_place_medal:

**Pull request merged**